main.rb

```diff
@@ -0,0 +1,6 @@
+require './app'
+def main
+  app = App.new
+  app.run
```

Comment:
- [ ] Kindly ensure your app works. There is no method for running your app. As soon as I run `ruby main.rb`, the app crashes. Kindly look carefully at the video you are to emulate and ensure your app works like that.

  ![image](https://github.com/Ashnaali3255/OOP-School-Library/assets/36542986/f53948c4-674f-4a20-a3dc-5c190890e8fc)